tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -203,8 +203,7 @@ def unpack_args(f: NativeFunction) -> Tuple[List[str], List[Binding]]:
 def get_base_name(f: NativeFunction) -> str:
     return f.func.name.name.base  # TODO: should be str(f.func.name.name)?
 
-def get_view_info(fn: NativeFunctionWithDifferentiabilityInfo) -> Optional[str]:
-    f = fn.func
+def get_view_info(f: NativeFunction) -> Optional[str]:
```

Comment:
I probably should have factored this into a separate PR, but this was just a small change to allow me to re-use `get_view_info` in the functionalization codegen. The extra DifferentiabilityInfo isn't used anywhere.

Reply:
never to late to stak